src/CMakeLists.txt

```diff
@@ -165,6 +174,9 @@ elseif(WIN32)
     set(PXCORE_FILES win/pxBufferNative.cpp win/pxOffscreenNative.cpp win/pxWindowNative.cpp
             win/pxEventLoopNative.cpp win/pxTimerNative.cpp win/pxClipboardNative.cpp pxViewWindow.cpp
             win/rtMutexNative.cpp win/rtThreadPoolNative.cpp)
+	if (SUPPORT_DUKTAPE)
+      set(PXCORE_FILES ${PXCORE_FILES} rtScriptDuk/debugger/duk_debugger_interface_windows.c)
```

Comment:
Let's create two subdirectories under rtScriptDuk/debugger/.  One should be called win the other should be called unix (following the same pattern as platform specific files in pxCore/src).  Let's have both files named the same (duk_debugger_interface.c) and put the unix version under rtScriptDuk/debugger/unix/ and the windows version under rtScriptDuk/debugger/win/.